### PR TITLE
revise acceptance.md helper arguments examples

### PR DESCRIPTION
**Pull request by nruth**
I just used this for the first time, and the guide is great, but it's not clear right now whether assert is injected, required, etc. I think changing the examples a little will help.

### CODE REVIEW

File: source/localizable/testing/acceptance.md [hunk 1]
```diff
@@ -170,7 +170,7 @@ for it to finish before running.
 [11]: http://emberjs.com/api/classes/Ember.Test.html#method_registerHelper
 
 The helper method will always be called with the current Application as the
-first parameter. Other parameters need to be provided when calling the helper. Helpers need to be registered prior to calling
+first parameter. Other parameters (such as assert) need to be provided when calling the helper. Helpers need to be registered prior to calling
```
Comment (locks) [inline source/localizable/testing/acceptance.md]: can you `, such as assert,` instead? thank you :)

Comment (nruth) [inline source/localizable/testing/acceptance.md]: yep, thanks for the feedback

**Comment by nruth** [comment]
Should I rebase, or do you do squashed merges?

**Comment by locks** [comment]
Please squash, as homu doesn't always properly squash them for us.

**Comment by locks** [comment]
Thanks for your contribution!

@homu r+

**Comment by nruth** [comment]
You're welcome.

**Pull request merged**